### PR TITLE
fix book image banner

### LINKED ISSUE #1215: the "IBM Cloud" is almost invisible
On the introduction page under "Kubernetes Cluster API Provider IBM Cloud" the words "IBM Cloud" is almost invisible beside the Kubernetes and IBM Cloud logo.
Screenshot (51)

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
**What this PR does / why we need it**:
- fix book image banner

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1215 

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
fix book image banner
```

### CODE REVIEW

File: docs/book/src/introduction.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Kubernetes Cluster API Provider IBM Cloud
 
 <p align="center">
-<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  width="100x"> 
-<img src="../images/ibm-cloud.svg" height="100x" width="200x" alt="IBM Cloud">
```
Comment (mkumatag) [inline docs/book/src/introduction.md:5]: can we delete this above `../images/ibm-cloud.svg` if not used anymore?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: docs/book/src/introduction.md (hunk 1 above)
Comment (Prajyot-Parab) [reply docs/book/src/introduction.md:5]: Done.

**Pull request merged**